modules/Quiz/resources/quiz.js

```diff
@@ -790,9 +790,25 @@
             if($.cpObject.cpArray[selectedQuestion].questionType === this.KIVQModule.QUESTIONS_TYPE.OPEN_QUESTION 
                 && $.cpObject.cpArray[selectedQuestion].openAnswer){
                 var text = $.cpObject.cpArray[selectedQuestion].openAnswer;
+                var feedback = $.cpObject.cpArray[selectedQuestion].feedback;
                 text = text.replace(/\n/g,"<br/>");
                 $(".yourAnswer").html(text).addClass("open-question-answer");
                 $(".correctAnswerText").html("");
+                if (feedback) {
+                    console.log($.cpObject)
+                    $(".feedback").html("Feedback").bind('click', function () {
+                        $(".feedback-modal").show();
+                        $(".reviewAnswerPlace").hide();
+                        $(".reviewAnswerFooter").hide();
```

Comment:
$(".feedback").html("Feedback").off('click') or use unbind

Reply:
$(".feedback").html("Feedback").off().bind(....) // or maybe it requires on("mouseClick")